src/libp2p_transport_tcp.erl

```diff
@@ -39,10 +39,11 @@ start_listener(Sup, Addr, TID) ->
             Options = libp2p_config:get_config(?CONFIG_SECTION, OptionDefaults),
             {TransportOpts, ListenOpts0} =
                 lists:partition(fun({Key, _}) ->
-                                        sets:is_element(Key, TransportKeys)
+                                        sets:is_element(Key, TransportKeys);
+                                   (_) -> true
```

Comment:
Can this be removed? What other options are you throwing in there?

Reply:
The `{raw, ...` option is not a 2-tuple.

Reply:
Right the raw option is added below this partition not before?